products/microos/main.pm

```diff
@@ -91,6 +91,15 @@ sub load_installation_tests {
     }
 }
 
+sub load_security_tests_selinux {
+    # SELinux tests for Micro OS operating system
+    # ALWAYS run following tests in sequence because of the dependencies
+    loadtest 'security/selinux/sestatus';
```

Comment:
sestatus.pm changes the enforcing mode of selinux as part of the test run.

This is totally negative behaviour in our case as we want to test SELinux as configured by selinux-targeted-setup installed by default on MicroOS.

I think it would be good to edit/reuse that test, but it needs changes to lines https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/security/selinux/sestatus.pm#L34-L46 to NOT hard-code configuration to SELinux (we want to test what the distro has configured)

Lines https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/security/selinux/sestatus.pm#L34-L46 will also need adaption given the MicroOS defaults are different than those used by this test

To prevent too much impact on other places which may be using that test, I'd recommend wrapping those changes in `if is_microos` statements which will require `use version_utils qw (...)` to include `is_microos`